src/pages/iou/request/step/IOURequestStepScan/index.native.tsx

```diff
@@ -466,7 +467,7 @@ function IOURequestStepScan({
         >
             {pdfFile && (
                 <PDFThumbnail
-                    style={{height: 1, width: 1, opacity: 0, position: 'absolute'}}
+                    style={styles.invisiblePdf}
```

Comment:
```suggestion
                    style={styles.invisiblePDF}
```
Thank you! Minor change, could you use capitals for pdf here?